src/containers/home-backedby/Homebackedby.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import './homebackedby.css'
 
 function Homebackedby() {
-    const organisationLogos = ['/protocol_labs.png', 'long-hash-ventures.png']
+    const organisationLogos = ['protocol_labs.png', 'long-hash-ventures.png', 'Big_Brain.png', 'Fenbushi_Capital.png', 'Formless.png', 'mask_network.svg', 'NGC.png', '2Link.jpg']
```

Comment:
NGC ventures logo is incorrect, it shouldn't be ngc asset management

Reply:
done